src/modules/mavlink/mavlink_mission.cpp

```diff
@@ -397,6 +408,17 @@ MavlinkMissionManager::current_item_count()
 	return _count[_mission_type];
 }
 
+uint32_t
+MavlinkMissionManager::current_item_crc()
```

Comment:
I misunderstood this as some sort of CRC for the current mission item (e.g. waypoint or something). Maybe rename to "current_mission_type_crc" or similar?